js/ooyala_ssai.js

```diff
@@ -162,6 +162,11 @@ OO.Ads.manager(function(_, $)
       amc.addPlayerListener(amc.EVENTS.AD_VOLUME_CHANGED, _.bind(this.onAdVolumeChanged, this));
       amc.addPlayerListener(amc.EVENTS.MUTE_STATE_CHANGED, _.bind(this.onMuteStateChanged, this));
       amc.addPlayerListener(amc.EVENTS.PLAY_STARTED, _.bind(this.onPlayStarted, this));
+      amc.addPlayerListener(amc.EVENTS.CONTENT_TREE_FETCHED, _.bind(this.onContentTreeFetched, this));
+    };
+
+    this.onContentTreeFetched = function (event, content) {
+      currentOffset = content.duration;
```

Comment:
I believe we can add tests here?

Reply:
sure 🤥

Reply:
but it's a blocker
tests are coming in the next pr